migration/src/m20220101_000001_create_table.rs

```diff
@@ -57,7 +57,7 @@ impl MigrationTrait for Migration {
 enum User {
     Table,
     Id,
-    UserName,
+    Name,
```

Comment:
And why did you changed this? is that some clippy thing?

Reply:
It's also a warning from Clipper about naming conventions for consistency with Rust's guidelines